redash/handlers/queries.py

```diff
@@ -49,11 +80,14 @@ def get(self):
 
         include_drafts = request.args.get('include_drafts') is not None
 
-        return [q.to_dict(with_last_modified_by=False)
-                for q in models.Query.search(term,
-                                             self.current_user.group_ids,
-                                             include_drafts=include_drafts,
-                                             limit=None)]
+        # this redirects to the new query list API that is aware of search
+        new_location = url_for(
+            'queries',
+            org_slug=current_org.slug,
+            search=term,
+            drafts='true' if include_drafts else 'false'
+        )
+        return {}, 301, {'Location': new_location}
```

Comment:
We always deploy API and client side together, so maybe we can just remove this?

Reply:
Ah, that'd be nicer indeed. And you don't make any guarantees regarding the API stability?

Reply:
Not really 🙄

I wonder if there are people who are using this API endpoint not in the application? 🤔